src/extensionFunctions.ts

```diff
@@ -34,6 +34,7 @@ import { getYamlContent } from './yamlUtils';
 const tmp = require('tmp');
 const fetch = require('node-fetch');
 const url = require('url-parse');
+const download = require('download');
```

Comment:
webpack was giving this error when import was using for this library:
`TS2349: This expression is not callable.`